manifests/config.pp

```diff
@@ -57,7 +56,7 @@
     # a complete relabeling is required when switching from disabled to
     # permissive or enforcing. Ensure the autorelabel trigger file is created.
     if $_real_mode in ['enforcing','permissive'] and
-      !$facts['os']['selinux']['enabled'] {
+    !$facts['os']['selinux']['enabled'] {
```

Comment:
This looks ugly. Perhaps it needs braces so the lint plugin understands it?

Reply:
fixed it